apps/web/src/specs/features/shared/profile-link.spec.tsx

```
@@ -0,0 +1,87 @@
+        <span>Alice</span>
+      </ProfileLink>
+    );
+
+    const link = screen.getByRole("link");
+    expect(link).toHaveAttribute("href", "/@alice");
+  });
+
+  it("renders the provided children inside the link", () => {
+    render(
+      <ProfileLink username="bob">
+        <span data-testid="child">Bob's avatar</span>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)